app/repositories/unko.py

```diff
@@ -28,7 +28,10 @@ def get_unko(unko_id: int) -> dict:
         Returns:
             dict: 指定したIDに対応するうんこオブジェクトの辞書。
         """
-        return Unko.query.filter(Unko.id == unko_id).first().to_dict()
+        unko = Unko.query.filter(Unko.id == unko_id).first()
+        if unko is None:
+            return None  # データが存在しない場合はNoneを返す
+        return unko.to_dict()
```

Comment:
(修正不要)このように簡潔に書くこともできます。

```suggestion
        return unko.to_dict() if unko else None
```